src/main/webapp/WEB-INF/tags/instructor/course/addCoursePanel.tag

```diff
@@ -0,0 +1,43 @@
+<%@ tag description="Add New Course Panel of Course Page" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ attribute name="googleId" type="java.lang.String" required="true" %>
+<%@ attribute name="courseIdToShow" type="java.lang.String" required="true" %>
+<%@ attribute name="courseNameToShow" type="java.lang.String" required="true" %>
```

Comment:
I believe the default interpretation of attributes is String, so you shouldn't need to specify the type.
I'm not entirely sure, though, so keep them if necessary.